aws/data_source_aws_ami_test.go

```diff
@@ -91,6 +94,9 @@ func TestAccAWSAmiDataSource_windowsInstance(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "state_reason.message", "UNSET"),
 					resource.TestCheckResourceAttr(resourceName, "tags.%", "0"),
 					resource.TestCheckResourceAttr(resourceName, "virtualization_type", "hvm"),
+					resource.TestCheckResourceAttr(resourceName, "platform_details", "Windows"),
```

Comment:
Sigh, data sources!

```
=== CONT  TestAccAWSAmiDataSource_windowsInstance
    data_source_aws_ami_test.go:65: Step 1/1 error: Check failed: Check 26/28 error: data.aws_ami.windows_ami: Attribute 'platform_details' expected "Windows", got "Windows with SQL Server Enterprise"
--- FAIL: TestAccAWSAmiDataSource_windowsInstance (9.13s)
```

Going to relax it for now to anything containing `Windows`.

```suggestion
					resource.TestMatchResourceAttr(resourceName, "platform_details", regexp.MustCompile(`Windows`)),
```